frontend/src/layout/navigation/SideBar/SideBarApps.tsx

```diff
@@ -0,0 +1,69 @@
+import { IconExtension } from 'lib/components/icons'
+import { urls } from 'scenes/urls'
+import { Scene } from 'scenes/sceneTypes'
+import { canInstallPlugins } from 'scenes/plugins/access'
+import { LemonButton } from 'lib/components/LemonButton'
+import { PluginSource } from 'scenes/plugins/source/PluginSource'
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { organizationLogic } from 'scenes/organizationLogic'
+import { navigationLogic } from '~/layout/navigation/navigationLogic'
+import { frontendAppsLogic } from 'scenes/apps/frontendAppsLogic'
+import { router } from 'kea-router'
+import { PageButton } from '~/layout/navigation/SideBar/PageButton'
+
+export function SideBarApps(): JSX.Element {
+    const { currentOrganization } = useValues(organizationLogic)
+    const { hideSideBarMobile, openAppSourceEditor, closeAppSourceEditor, setOpenAppMenu } = useActions(navigationLogic)
+    const { appSourceEditor, openAppMenu } = useValues(navigationLogic)
+    const { frontendApps } = useValues(frontendAppsLogic)
+    const { currentLocation } = useValues(router)
+
+    return (
+        <>
+            {Object.values(frontendApps).map(({ id, pluginId, title }) => (
+                <PageButton
+                    key={id}
+                    icon={<IconExtension />}
+                    title={title || `App #${id}`}
+                    identifier={currentLocation.pathname === urls.frontendApp(id) ? Scene.FrontendAppScene : 'nope'}
+                    to={urls.frontendApp(id)}
+                    sideAction={
+                        canInstallPlugins(currentOrganization)
+                            ? {
+                                  identifier: 'app-menu',
+                                  onClick: () => setOpenAppMenu(openAppMenu === id ? null : id),
+                                  popup: {
```

Comment:
I think yes. I also think having a lot of edit icons below one another makes for a very strong visual "look at me" signal that we want to avoid. A subtle menu is better.

Some things we can add in the menu:

- Unpin from sidebar (and move to a "browse apps" dropdown menu, similar to "pinned" dashboards now)
- Uninstall
- Configure